meinberlin/apps/activities/views.py

```diff
@@ -25,22 +24,13 @@ def form_valid(self, form):
         return super().form_valid(form)
 
     def get_object(self, queryset=None):
-        try:
-            first_activity = models.Activity.objects \
-                .filter(module=self.module) \
-                .first()
-            return first_activity
-        except AttributeError:
-            return None
+        return models.Activity.objects.filter(module=self.module).first()
```

Comment:
Why is this still here? I thought you said it was needed after all?

Reply:
What was not needed is the try except clause, since there is no occasion where an AttributeError is raised. As we tested together .first() already returns None when there are no entries, so it's not needed to handle this case separately.... hmmm so from the behaviour the code that is there now should do exacly the same, I only removed the try except clause and the result is directly returned.

Reply:
Ah alright, than I misunderstood you.